loopchain/blockchain/blocks/v0_4/block_verifier.py

```diff
@@ -0,0 +1,211 @@
+# Copyright 2018-current ICON Foundation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""block verifier for version 0.4 block"""
+
+from typing import TYPE_CHECKING, Callable, Sequence
+
+from loopchain import configure as conf
+from loopchain.blockchain.blocks import BlockVerifier as BaseBlockVerifier
+from loopchain.blockchain.blocks.v0_4 import BlockHeader, BlockBody, BlockBuilder
+from loopchain.blockchain.exception import NotInReps
+from loopchain.blockchain.types import ExternalAddress, Hash32
+from loopchain.blockchain.votes.v0_4 import BlockVotes, LeaderVotes
+
+if TYPE_CHECKING:
+    from loopchain.blockchain.blocks import Block
+
+
+class BlockVerifier(BaseBlockVerifier):
+    version = BlockHeader.version
+
+    def verify(self, block: 'Block', prev_block: 'Block', blockchain=None, **kwargs):
+        super().verify(block, prev_block, blockchain, **kwargs)
+
+    def verify_loosely(self, block: 'Block', prev_block: 'Block', blockchain=None, **kwargs):
+        super().verify_loosely(block, prev_block, blockchain, **kwargs)
```

Comment:
I think those are redundant lines. Can you explain that?

Reply:
ok. good point!